src/components/Composer/index.tsx

```diff
@@ -458,7 +378,7 @@ function Composer({
 
                         textInput.current.focus();
                     });
-                    if (props.onFocus) {
+                    if (props?.onFocus) {
```

Comment:
why props is optional here? Won't this work? - `props.onFocus?.(e);`

Reply:
Right, changed it